### PR TITLE
Fixed Existing Tamil Language Books file

### PR DESCRIPTION
**Pull request by Ratheshan03**
## What does this PR do?
Improved the overall markdown file by updating the author and section name, and removed the unwanted spaces and symbols

### Description
Removed unwanted whitespaces, edited correct author names and updated the section name

### Why is this valuable (or not)?
Made it more clear and accurate

### How do we know it's really free?
https://kaniyam.com/ebooks/
check the website to get to know about the author names, and section

### For book lists, is it a book? For course lists, is it a course? etc.
Its a Code fix update

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-ta.md [hunk 1]
```diff
@@ -26,6 +26,11 @@
 * [எளிய தமிழில் VR/AR/MR](https://freetamilebooks.com/ebooks/vr_ar_mr/) - இரா.அசோகன் (PDF)
 
 
+### Big Data
+
+* [எளிய தமிழில் Big Data](https://freetamilebooks.com/ebooks/learn-bigdata-in-tamil) - து.நித்யா (PDF)
```
Comment (eshellman) [inline books/free-programming-books-ta.md:31]: In general, we do not link to file sharing sites, or file hosting sites such as google docs. or dropbox.
It's usually not helpful to add a category for a single resource. Leave this in hadoop

**Review by eshellman — CHANGES_REQUESTED** [review]
It doesn't help people find things if the category just recapitulates the title.

File: books/free-programming-books-ta.md (hunk 1 above)
Comment (Ratheshan03) [reply books/free-programming-books-ta.md:31]: okay sure!

Comment (Ratheshan03) [reply books/free-programming-books-ta.md]: @eshellman requested changes have been committed!

**Pull request merged**